llvm/lib/Target/AArch64/AArch64Processors.td

```diff
@@ -723,6 +746,9 @@ def ProcessorFeatures {
                                  FeaturePerfMon, FeatureETE, FeatureTRBE,
                                  FeatureSPE, FeatureMTE, FeatureSVE2BitPerm,
                                  FeatureFP16FML, FeatureSPE_EEF];
+  list<SubtargetFeature> X925 = [HasV9_2aOps, FeatureMTE, FeatureFP16FML,
```

Comment:
Should this include FeatureSVE2BitPerm? It is included in the AEK_ list, and X4 the features.

Same for the A725 features.

Reply:
Thanks, good spot. Fixed.